### PR TITLE
Pulling DB & Files From Pantheon

### PR DESCRIPTION
**Pull request by amitaibu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -171,3 +172,25 @@ In order to deploy upon every merge automatically by Travis, you shall:
 
 Optionally you can specify which target branch you'd like to push on Pantheon, by default it's `master`, so the target is the DEV environment, but alternatively you can issue:
 `ddev deploy:config-autodeploy [your new token] [pantheon project name] [gh_branch] [pantheon_branch]`
+
+## Pulling DB & Files From Pantheon
+
+### First Time
+
+To set the Pantheon environment to be pulled from to `LIVE`, execute
```
Comment (AronNovak) [inline README.md:180]: The sentence is not finished here.

**Comment by amitaibu** [comment]
@AronNovak could you please check the failing test `\Drupal\Tests\server_general\ExistingSite\ServerGeneralSearchTest::testBasicIndexing`

**Comment by AronNovak** [comment]
Checking now, normally https://github.com/Gizra/drupal-starter/pull/119 fixed it recently.

**Comment by AronNovak** [comment]
@amitaibu It's fixed now.

**Comment by amitaibu** [comment]
Thanks

**Pull request merged**